mne/datasets/utils.py

```diff
@@ -95,7 +95,7 @@ def _get_path(path, key, name):
         return path
     # 2. get_config(key)
     # 3. get_config('MNE_DATA')
-    path = get_config(key, get_config('MNE_DATA'))
+    path = get_config(key or 'MNE_DATA')
```

Comment:
I don't quite get this one. It means that before if the config for the given key did not exist, we would use `MNE_DATA`, but now we only do so if `key is None`? This seems like an incorrect change in behavior?

Reply:
`None` is a sentinel value for `get_config()` that returns all config keys and their values that are defined in the config file.  But here, `key` being `None` actually means something different (viz, "this dataset is MNE-external and is not kept track of in the config")

Reply:
so this change makes it so that, when `key is None`, we get the config value for `"MNE_DATA"` instead of a big dict of all defined config values.

Reply:
put another way, it updates the `_get_path()` function to gracefully handle `key=None` input, which previously it was not designed to handle.

Reply:
Ahh that makes sense, thanks!

Reply:
Actually you're right, maybe this should still include the fallback of `get_config('MNE_DATA')` for cases of non-empty strings that aren't valid config keys?

Reply:
Or valid ones that aren't set yet in the config, which is pretty common -- people just set MNE_DATA once then all downloads just automatically go to that location